libvirt/tests/src/sriov/locked_memory_check/sriov_with_hotplug_memory.py

```diff
@@ -105,7 +105,7 @@ def hotplug_mem_device(vm_name, mem_dict, current_mem):
         test.log.info("TEST_STEP: Start the vm and check locked memory limit.")
         vm.start()
         test.log.debug("The current guest xml is:%s", virsh.dumpxml(vm_name).stdout_text)
-        vm_session = vm.wait_for_serial_login(timeout=240).close
+        vm.wait_for_login(timeout=240).close
```

Comment:
Please confirm again whether it needs to be changed to the following, like my previous comment, you need to call close().
```suggestion
        vm.wait_for_login(timeout=240).close()
```